source/css/sass/patterns/organisms/content-header.scss

```diff
@@ -18,7 +18,7 @@
     content: "";
     display: block;
     @include padding($baselinegrid-space-small - 1, "block-start");
-    width: 100%;
+    @include block-size(100%);
```

Comment:
`inline-size`

Reply:
(https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request#applying-a-suggested-change)